# HG changeset patch
# User jwe
# Date 1069503567 0
#      Sat Nov 22 12:19:27 2003 +0000
# Node ID 7604168d3e2af15629e1a79029fce37248f7f636
# Parent  2d237a76dbdd090368b2652868fc05068fb6c2e5
[project @ 2003-11-22 12:19:27 by jwe]
ov-builtin.h

diff --git a/src/ov-dld-fcn.h b/src/ov-dld-fcn.h
--- a/src/ov-dld-fcn.h
+++ b/src/ov-dld-fcn.h
@@ -42,16 +42,18 @@ class octave_value_list;
 
 // Dynamically-linked functions.
 
 class
 octave_dld_function : public octave_builtin
 {
 public:
 
+  octave_dld_function (void);
+
   octave_dld_function (octave_builtin::fcn ff, const octave_shlib& shl,
 		       const std::string& nm = std::string (),
 		       const std::string& ds = std::string ());
 
   ~octave_dld_function (void);
 
   void mark_fcn_file_up_to_date (const octave_time& t) { t_checked = t; }
 
@@ -64,18 +66,16 @@ public:
   bool is_system_fcn_file (void) const { return system_fcn_file; }
 
   bool is_builtin_function (void) const { return false; }
 
   bool is_dld_function (void) const { return true; }
 
 private:
 
-  octave_dld_function (void);
-
   octave_dld_function (const octave_dld_function& m);
 
   octave_shlib sh_lib;
 
   // The time the file was last checked to see if it needs to be
   // parsed again.
   mutable octave_time t_checked;
 
